app/views/placements/support/providers/users/index.html.erb

```diff
@@ -12,22 +12,12 @@
       <h2 class="govuk-heading-m"><%= t(".heading") %></h2>
       <%= govuk_button_to(t(".add_user"), new_placements_support_provider_user_path(@provider), method: :get) %>
 
-      <%= govuk_table do |table| %>
-        <% table.with_head do |head| %>
-          <% head.with_row do |row| %>
-            <% row.with_cell(header: true, text: t(".attributes.users.name")) %>
-            <% row.with_cell(header: true, text: t(".attributes.users.email")) %>
-          <% end %>
-        <% end %>
-        <% table.with_body do |body| %>
-          <% @users.each do |user| %>
-            <% body.with_row do |row| %>
-              <% row.with_cell(text: govuk_link_to(user.full_name, placements_support_provider_user_path(id: user.id))) %>
-              <% row.with_cell(text: user.email) %>
-            <% end %>
-          <% end %>
-        <% end %>
-      <% end %>
+      <%= render partial: "shared/users_list", locals: {
```

Comment:
Not a blocker, but can you delete the unused translations (`.attributes.users.email`)?